src/material-design-lite.scss

```diff
@@ -51,3 +51,4 @@
 @import "tooltip/tooltip";
 @import "shadow/shadow";
 @import "grid/grid";
+@import "chip/chip";
```

Comment:
Components should all be alphabetical *except* for shadow/grid which come last for maximum specificity.